models/xero__invoice_line_items.sql

```diff
@@ -1,3 +1,8 @@
+{%- set using_tracking_categories = (
+    var('xero__using_journal_line_has_tracking_category', True)
```

Comment:
Why are we using the journal line variable here? Shouldn't this be the invoice line variable?

Reply:
Yes. Flipped over.